--- a/target/linux/qualcommbe/image/ipq95xx.mk
+++ b/target/linux/qualcommbe/image/ipq95xx.mk
@@ -8,11 +8,11 @@ define Build/be7000-append-ubi
 		echo "vol_type=dynamic"; \
 		echo "vol_name=kernel"; \
 		echo "image=$(IMAGE_KERNEL)"; \
-		echo "[ubi_rootfs]"; \
+		echo "[rootfs]"; \
 		echo "mode=ubi"; \
 		echo "vol_id=1"; \
 		echo "vol_type=dynamic"; \
-		echo "vol_name=ubi_rootfs"; \
+		echo "vol_name=rootfs"; \
 		echo "image=$(IMAGE_ROOTFS)"; \
 		echo "vol_size=$$rootsize"; \
 		echo "[rootfs_data]"; \
